### DISCUSSION TITLE
Do you have package on pub.dev?

### DISCUSSION THREAD

**Discussion by PrzemyslawPluszowy** [isar-community/isar#59] [Q&A] [answered]
You have or planning put package to pub dev?

**Comment by vicenterusso** [comment]
Yes, eventually.

We already have a profile on `pub.dev`, but the main problem is that we need to rename all packages, and all users will have to do that too.

We don’t know if Simon will return (we really hope that he does). If he does, all users would have to rename imports back.

If we publish on `pub.dev`, we will be officially detached from `isar/isar`.

So, personally, I would give it some more time before we make that move.

**Comment by ibrahimdevs** [comment]
@vicenterusso when the time is up? I'm worried about to continue with isar. But I think it is time to publish a seperate package completely backward compatible with Isar, users should only make rename. 

Great work, thanks 🙏

**Reply by vicenterusso** [reply, marked as answer]
I believe its the right time! Already working on it
